src/domain/roadsignactuator.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+import json
+
+from domain.actuator import Actuator
+
+
+class RoadSignActuator(Actuator):
+    """RoadSignActuator, specialization of Actuator."""
+
+    class Commands(Enum):
+        display_on = json.dumps({"display": True})
```

Comment:
As per discussion with Cesare on Notifier PR, we will set all Enum values to UPPERCASE as coding style.